test/Microsoft.TestPlatform.AcceptanceTests/Extension/RunnnerInfo.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.TestPlatform.AcceptanceTests;
 /// <param name="RunnerFramework"></param>
 /// <param name="TargetFramework"></param>
 /// <param name="InIsolationValue">Supported value = <c>/InIsolation</c>.</param>
-public record RunnerInfo(string RunnerFramework, string TargetFramework, string InIsolationValue = "")
+public record struct RunnerInfo(string RunnerFramework, string TargetFramework, string InIsolationValue = "")
```

Comment:
pls add a comment if it's for serialization reason, I would remove it as we have two strings on the heap so struct doesn't make a lot of sense as-is